### ISSUE TITLE
CSV (or some other) import of data from my Crane scales

### ISSUE THREAD

**Issue by LimeBlast** [kochka/WeightLogger#23]
Hi, sorry if this is the wrong place for this, but I'm hoping someone might be able to help me figure something out.

I've got a [Crane Diagnostic Scale](https://www.cranesportsconnect.com/) which I've been using for a couple of months now. This automatically syncs with an app on my phone, which in turn stores all the data in the cloud (which I'm able to access from their site, and download as a CSV, etc).

I've just discovered WeightLogger, and am interested in being able to export all the data from the site into WeightLogger, for that to then be synced with Garmin.

Does anyone know how I might be able to do this? Thank you.

**Comment by kochka** [comment]
I've never done any CSV import mostly because of units. Furthermore it need a well defined format.
You can still do it with some IT knowledge by exporting the database via setting (backup) and edit it (SQLite), then reimport it...
If you want, I can do it for you. Send me your CSV and I will send you back a backup (I've some scripts for that).

**Comment by LimeBlast** [comment]
That's very generous of you, but if it's just an SQLite database, I should be able to edit it myself (that said, I've just tried a backup, but am unable to find the export folder, or the backfile anywhere on the file system).

Thank you.

**Comment by kochka** [comment]
There are a setting to choose export path. Default should be a weightlogger directory in user's root directory.

**Comment by kochka** [comment]
Just seen in your profile you know ruby so I give you the little script I've done to fill a backup with CSV.
It's super simple but still will make you save 5 min ;)
https://gist.github.com/kochka/f4b07b5499ef1111cfe7#file-import-rb

**Comment by LimeBlast** [comment]
Oh excellent, thank you. I've also found the .bkp file and am just taking a look at it now.

My ultimate goal would be to have something which automatically scrapes the crane site for new measurements and automatically submits them to Garmin (either via your app, or using the same technique as you're using) - but I think that's a project for a rainy weekend.

I've got enough for now, so I'm going to close the ticket. Thank you for your help.
